appengine/pond_storage/opponents.py

```diff
@@ -19,27 +19,115 @@
 
 import cgi
 import json
+import math
+import random
 from google.appengine.ext import ndb
 from pond_storage import *
 
-def get_opponent_ducks(duck):
-  """Returns list of opponent ducks."""
-  # TODO: Add filter for published ducks
-  # TODO: Add logic here to pick the duck based on the ranking
-  user_key = get_user_key(users.get_current_user())
-  duck_query = Duck.query()
-  duck_list = []
-  for duck in duck_query:
-    if duck.key.parent() != user_key:
-      duck_list.append(get_duck_info(duck))
-  return duck_list[0:3]
+START_PERCENT = .10
+END_PERCENT = .15
+NEARBY_PERCENT = .10
```

Comment:
Please add comments to clarify what these values are or I think they aren't adding much as constants since they are only used once (to move to methods that use them).

Reply:
Added comments. According to this https://www.python.org/dev/peps/pep-0008/#constants are usually defined at the module level. I added _ in front since they are for internal